src/platform/bouffalolab/common/CHIPDevicePlatformConfig.h

```diff
@@ -48,6 +52,10 @@
 #define CONFIG_BOUFFALOLAB_FACTORY_DATA_ENABLE 0
 #endif
 
+#ifndef CONFIG_ENABLE_CHIP_SHELL
+#define CONFIG_ENABLE_CHIP_SHELL 0
+#endif
```

Comment:
It's odd to have this in a core platform config, if it's only used in examples.  But OK.

Reply:
CONFIG_ENABLE_CHIP_SHELL is configured in example BUILD.gn file and removed CONFIG_ENABLE_CHIP_SHELL configuration from [CHIPDevicePlatformConfig.h](https://github.com/project-chip/connectedhomeip/pull/29756/files/13dc02c073f3c9b933f6906f846baf4b30b40f1c#diff-68b16af38589f80248ae4e82fab1ffb74a51288fe47f22860810b47f8b6ffe27)